atommic/collections/multitask/rs/nn/base.py

```diff
@@ -155,8 +157,10 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):  # noqa: MC0001
         self.log_multiple_modalities = cfg_dict.get("log_multiple_modalities", False)
 
         # Set threshold for segmentation classes. If None, no thresholding is applied.
+        self.segmentation_type = cfg_dict.get("segmentation_type", "MLS")
```

Comment:
@TimPaquaij maybe it's obvious, but now self.segmentation_type = cfg_dict.get("segmentation_type", "MLS") why do we need to set it to mls by default?